components/deployhq/sources/common.mjs

```diff
@@ -0,0 +1,34 @@
+import { createVerify } from "crypto";
+
+export const STATUS_RUNNING = "running";
+export const STATUS_COMPLETE = "completed";
+export const STATUS_FAILED = "failed";
+
+const DEPLOYHQ_KEY = `
+-----BEGIN PUBLIC KEY-----
+MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDXJcP2N6NtcN26Q8nVaidXOA0w
+RxWK2HQTblIaQdGRDjqTvhrSlFuV5N4jz7w/w8uskP20G7ZQ+CkHwIXrWk76KZJn
+pdoOHPO6AqRmEFgV5Q6Y1CR77mvnT9O21hTnfzfyyiAdQC2oO8M9/jeLRPTAqmkG
+xdQa8iepUz4BwrrHmwIDAQAB
+-----END PUBLIC KEY-----`;
```

Comment:
Is this key different for each user?

Reply:
No, it's global.

https://www.deployhq.com/support/integrations/http-post#our-public-key